.github/workflows/ruskwallet_ci.yml

```diff
@@ -25,60 +25,8 @@ jobs:
         with:
           filters: |
             run-ci:
-              - 'wallet-core/**'
```

Comment:
These were here because if any changes to `wallet-core` or any dependency of rusk-wallet is done then rusk-wallet should recompile and be updated with it else the CI will break

Reply:
I think this is fair enough, kept the wallet-core and execution-core dir change detection.